common/model/db/sql/sqlSelectBuilder.js

```diff
@@ -0,0 +1,83 @@
+import * as A from '@core/arena'
+import SqlBuilder from '@common/model/db/sql/sqlBuilder'
+
+class SqlSelectBuilder extends SqlBuilder {
+  constructor() {
+    super()
+    this._select = []
+    this._from = []
+    this._where = []
+    this._groupBy = []
+    this._offset = null
+    this._limit = null
+    this._orderBy = []
+    this._queryParams = {}
```

Comment:
this can be called _params. and I would move up this and related methods (to the parent class).

Reply:
ok